src/main/java/seedu/mindmymoney/constants/ExpenditureFields.java

```diff
@@ -0,0 +1,12 @@
+package seedu.mindmymoney.constants;
+
+/**
+ * Container for fields in expenditure.
+ */
+public enum ExpenditureFields {
+    expenditure,
+    category,
+    description,
+    amount,
+    time,
```

Comment:
Enumerations fields should be in capital letters as seen in https://nus-cs2113-ay2122s2.github.io/website/schedule/week4/topics.html#w4-4-oop-java-enumerations

Reply:
thanks!!